test/core/V3/prohibition/GenArt721CoreV3_Integration_PROHIBITION.test.ts

```diff
@@ -0,0 +1,722 @@
+          .connect(config.accounts.artist)
+          .toggleProjectIsActive(config.projectZero);
+      });
+
+      it("behaves as expected when toggling project activation when user is approved for selector", async function () {
+        const config = await loadFixture(_beforeEach);
+        await config.adminACL
+          .connect(config.accounts.deployer)
+          .toggleContractSelectorApproval(
+            config.genArt721Core.address,
+            config.genArt721Core.interface.getSighash("toggleProjectIsActive"),
+            config.accounts.user.address
+          );
+        await config.genArt721Core
+          .connect(config.accounts.user)
+          .toggleProjectIsActive(config.projectZero);
+      });
+
+      it("reverts when toggling project activation from unverified account", async function () {
+        const config = await loadFixture(_beforeEach);
+        await expectRevert(
+          config.genArt721Core
+            .connect(config.accounts.user)
+            .toggleProjectIsActive(config.projectZero),
+          "Only Admin ACL allowed"
+        );
+      });
+
+      it("reverts when toggling project from verified user that is not artist", async function () {
+        const config = await loadFixture(_beforeEach);
+        await config.adminACL
+          .connect(config.accounts.deployer)
+          .toggleContractArtistApproval(
+            config.genArt721Core.address,
+            config.accounts.artist2.address
+          );
+
+        await expectRevert(
+          config.genArt721Core
+            .connect(config.accounts.artist2)
+            .toggleProjectIsActive(config.projectZero),
+          "Only Admin ACL allowed"
+        );
+      });
+    });
+
+    describe("reverts on project locked", async function () {
+      const config = await loadFixture(_beforeEach);
+      it("reverts if try to add script", async function () {
+        const config = await loadFixture(_beforeEach);
+        await mintProjectUntilRemaining(
+          config,
+          config.projectZero,
+          config.accounts.artist,
+          0
+        );
+        // wait until project is locked
+        await advanceEVMByTime(FOUR_WEEKS + 1);
+        // expect revert
+        await expectRevert(
+          config.genArt721Core
+            .connect(config.accounts.artist)
+            .addProjectScript(config.projectZero, "lorem ipsum"),
+          "Only if unlocked"
+        );
+      });
+    });
+
+    describe("coreVersion", function () {
+      it("returns expected value", async function () {
+        const config = await loadFixture(_beforeEach);
+        let targetCoreVersion;
+        if (coreContractName === "GenArt721CoreV3") {
+          targetCoreVersion = "v3.0.2";
+        } else if (coreContractName === "GenArt721CoreV3_Explorations") {
+          targetCoreVersion = "v3.0.3";
+        } else if (coreContractName.includes("GenArt721CoreV3_Engine")) {
+          targetCoreVersion = "v3.1.2";
```

Comment:
this test is failing - can you update to handle the prohibition forked case (`v3.1.3`), thanks!

Reply:
I must be losing my mind. I could've sworn I did this... or was it somewhere else?

Reply:
done

Reply:
i think this test is redundant with one in the views tests, so I promise you aren't losing your mind 😅